src/Protocol/DFRN.php

```diff
@@ -2079,8 +2079,8 @@ private static function updateContent($current, $item, $importer, $entrytype)
 				return false;
 			}
 
-			$fields = ['title' => $item["title"], 'body' => $item["body"],
-					'tag' => $item["tag"], 'changed' => DateTimeFormat::utcNow(),
+			$fields = ['title' => defaults($item["title"], ''), 'body' => defaults($item["body"], ''),
```

Comment:
The `defaults()` syntax for array usually is `defaults($item, 'title', '')`, unless you are sure the keys exists and you can use the shorter ternary operator form `$item['title'] ? : ''`.

Reply:
According to the code ```$item['title']``` does exist. Although I don't really understand why this can be ```null```. It is the result of this: ```$item["title"] = $xpath->query("atom:title/text()", $entry)->item(0)->nodeValue;```.

Reply:
Then the short ternary operator form would be enough.